Leanplum-SDK/Classes/Internal/Leanplum.m

```diff
@@ -2399,6 +2399,7 @@ + (void)forceContentUpdate:(LeanplumVariablesChangedBlock)block
             ![messages isEqualToDictionary:[LPVarCache sharedCache].messageDiffs] ||
             ![updateRules isEqualToArray:[LPVarCache sharedCache].updateRulesDiffs] ||
             ![eventRules isEqualToArray:[LPVarCache sharedCache].eventRulesDiffs] ||
+            ![variants isEqualToArray:[LPVarCache sharedCache].variants] ||
```

Comment:
Can you give me quick context why we need this variants? Also, any reasons, we dont have test cases for this?

Reply:
The issue is that when calling force content update if only variants are updated it wont be saved until next time start is executed.

Reply:
Do we have existing test cases for forceContentUpdate method?